src/App.test.tsx

```diff
@@ -30,4 +32,98 @@ describe('App component:', () => {
       expect(screen.getByText(/emotions category/i)).toBeInTheDocument();
     });
   });
+  // ***************************************************INTEGRATION TESTS**********************************************
+  describe('opens', () => {
+    it('sidebar on header button click', () => {
+      const renderComp = render(
+        <MemoryRouter initialEntries={['/']}>
+          <App />
+        </MemoryRouter>
+      );
+      const openButton = renderComp.container.getElementsByClassName('SideBarOpenButton');
```

Comment:
Actually variable name doesn't represent what stored inside. `renderComp.container.getElementsByClassName` returns a collection 

```suggestion
      const { 0: openButton } = renderComp.container.getElementsByClassName('SideBarOpenButton');
```